src/test/resources/TestSuites/validationTests.xml

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
```

Comment:
kindly delete this file as it's not part of the project itself.

Reply:
I apologize for this. I was using it to check that the functionalities are intact. 😅